svthalia/concrexit
# Centralize event status logic and allow message overriding

Closes #1381
Closes #2600 



### Summary
Create a centralized place to determine registration status (e.g. cancelled, cancelled-late, open, closed)

### How to test
Steps to test the changes you made:
1. Create an event
2. Register/deregister/etc (all possible combinations)
3. Check that the text makes sense

I should still check that the addition to the API etc make sense. We should also look into removing the relevant logic from the app, and let it use the status (string etc) returned from the server.
